pkgs/top-level/aliases.nix

```
@@ -532,4 +532,9 @@ mapAliases ({
 
   # added 2019-10-28
   gnatsd = nats-server;
+
+  # added 2020-01-10
+  tor-browser-bundle = throw "use tor-browser-bundle-bin instead.";
+  # added 2020-01-10
+  tor-browser-unwrapped = throw "use tor-browser-bundle-bin instead.";
```

What if we set it up to issue a warning of some sort, and then automatically did the right thing? Also:

```suggestion
  tor-browser-unwrapped = throw "tor-browser-unwrapped was removed because it was out of date and inadequately maintained. Please use tor-browser-bundle-bin instead. See #77452.";
```

---

I want to avoid automatically doing "the right thing here", but having people explicitly making the decision. I will update the message however.